packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -9,15 +9,15 @@ import {
   useIsFocused,
   useRowStyle,
 } from '../../context';
-import { Collapse, Fade } from '@mui/material';
+import { Fade } from '@mui/material';
```

Comment:
😢 

sad so lose the collapse transition.. but it isn't possible without rendering all expando sections which is what we are trying to avoid.

Reply:
on the plus side it feel snappier :)